src/rebar_utils.erl

```diff
@@ -158,11 +158,21 @@ sh(Command0, Options0) ->
     ?DEBUG("Port Cmd: ~s\nPort Opts: ~p\n", [Command, PortSettings]),
     Port = open_port({spawn, Command}, PortSettings),
 
-    case sh_loop(Port, OutputHandler, []) of
-        {ok, _Output} = Ok ->
-            Ok;
-        {error, {_Rc, _Output}=Err} ->
-            ErrorHandler(Command, Err)
+    case os:type() of
+        {win32, _} ->
```

Comment:
However, `rebar_utils:sh` could be used to run another things, not only robocopy. And I believe in this case such behaviour could be harmful. I think it is better to handle all possible robocopy exit codes on higher level, in code which calls `rebar_utils:sh`.